pendulum_control/src/pendulum_demo.cpp

```diff
@@ -264,6 +200,22 @@ int main(int argc, char * argv[])
       ("pendulum_sensor", qos_profile, controller_subscribe_callback,
       nullptr, false, state_msg_strategy);
 
+  // Create a lambda function to accept user input to command the pendulum
+  auto controller_command_callback =
+    [&pendulum_controller](const pendulum_msgs::msg::JointCommand::SharedPtr msg) -> void
```

Comment:
Hmm, I would prefer a `ConstSharedPtr` (SharedPtr to const message) but I ran into this issue:

https://github.com/ros2/rclcpp/issues/118

Ignoring the part about publishing const messages for now, this pull request is needed to get the message as a shared pointer to a const in the subscripton callback:

https://github.com/ros2/rclcpp/pull/119